x/feegrant/keeper/msg_server.go

```diff
@@ -84,3 +84,9 @@ func (k msgServer) RevokeAllowance(ctx context.Context, msg *feegrant.MsgRevokeA
 
 	return &feegrant.MsgRevokeAllowanceResponse{}, nil
 }
+
+// PruneAllowances removes expired allowances from the store.
+func (k msgServer) PruneAllowances(ctx context.Context, _ *feegrant.MsgPruneAllowances) (*feegrant.MsgPruneAllowancesResponse, error) {
+	err := k.RemoveExpiredAllowances(ctx, 5)
```

Comment:
Should this be a module parameter?

Reply:
i think its fine since fees area already so low that the user can call it a few times

Reply:
@tac0turtle you think we shouldn't have params? Already added them, but can take them out. I'm not 100% sure we should have them